packages/app/features/dashboard/profile/index.tsx

```diff
@@ -54,8 +55,9 @@ export function ProfileScreen() {
       </View>
       <ProfileRow
         icon={<Like size={24} color={tw.color("blue")} />}
-        number={3}
+        number={userLikesData?.userLikes?.length}
```

Comment:
what is displayed, when data is `undefined`?

Reply:
Nothing, the same as when you pass 0, `<Text className="text-sm">{number && number > 0 ? number : ""}</Text>`.
Probably can remove `&& number > 0` and it will be working the same

Reply:
okay, my suggestion is to rename it `trailingText` with a type string and an empty string as a default value.

Reply:
Ok, will do